inabox/tests/integration_v2_test.go

```diff
@@ -1,437 +1,247 @@
 package integration_test
 
 import (
-	"bytes"
 	"context"
 	"crypto/rand"
-	"fmt"
-	"math/big"
+	"encoding/hex"
 	"time"
 
-	"github.com/Layr-Labs/eigenda/api/clients/v2/relay"
-	"github.com/docker/go-units"
+	"github.com/Layr-Labs/eigenda/api/clients/codecs"
+	"github.com/Layr-Labs/eigenda/api/clients/v2/coretypes"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"golang.org/x/crypto/sha3"
 
 	"github.com/Layr-Labs/eigenda/api/clients/v2"
-	commonpb "github.com/Layr-Labs/eigenda/api/grpc/common/v2"
+
 	disperserpb "github.com/Layr-Labs/eigenda/api/grpc/disperser/v2"
-	verifierbindings "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDACertVerifierV2"
-	"github.com/Layr-Labs/eigenda/core"
 	auth "github.com/Layr-Labs/eigenda/core/auth/v2"
-	corev2 "github.com/Layr-Labs/eigenda/core/v2"
 	dispv2 "github.com/Layr-Labs/eigenda/disperser/common/v2"
-	"github.com/Layr-Labs/eigenda/encoding/utils/codec"
-	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"github.com/wealdtech/go-merkletree/v2"
-	"github.com/wealdtech/go-merkletree/v2/keccak256"
 )
 
 var _ = Describe("Inabox v2 Integration", func() {
 	/*
 		This end to end test ensures that:
-		1. a blob can be dispersed using the lower level disperser client to successfully produce a blob certificate
-		2. the blob certificate can be verified on chain using the EigenDACertVerifierV2 contract
+		1. a blob can be dispersed using the lower level disperser client to successfully produce a blob status response
+		2. the blob certificate can be verified on chain using the immutable static EigenDACertVerifier and EigenDACertVerifierRouter contracts
 		3. the blob can be retrieved from the disperser relay using the blob certificate
 		4. the blob can be retrieved from the DA validator network using the blob certificate
+		5. updates to the EigenDACertVerifierRouter contract can be made to add a new cert verifier with at a future activation block number
+		6. the new cert verifier will be used to verify the blob certificate at the future activation block number
 
+		TODO: Decompose this test into smaller tests that cover each of the above steps individually.
```

Comment:
agree with this. Think it might be nice to have a test which is purely core side, and another which covers all integration stuff (CertVerifier contract/router stuff)